app/assets/javascripts/discourse/mixins/name-validation.js.es6

```diff
@@ -3,6 +3,11 @@ import { default as computed } from 'ember-addons/ember-computed-decorators';
 
 export default Ember.Mixin.create({
 
+  @computed()
+  nameInstructions() {
+    return I18n.t(Discourse.SiteSettings.full_name_required ? 'user.name.instructions_required' : 'user.name.instructions');
```

Comment:
Instead of accessing the global `Discourse.SiteSettings`,  would `this.siteSettings` work instead?